### ISSUE TITLE
Thereom callout without annoying modals!

### ISSUE THREAD

**Issue by RyotaUshio** [RyotaUshio/obsidian-math-booster#185]
Time to stop this madness.

For example:

Numbered: (number='auto') 
```
> [!theorem|Uniform law of large numbers]
> Let $\mathcal{F}$ be a class of functions that ...
```

Unnumbered: (number='') 
```
> [!theorem|*Uniform law of large numbers]
> Let $\mathcal{F}$ be a class of functions that ...
```

Custom number (e.g. number='1.1'):
```
> [!theorem|1.1|Uniform law of large numbers]
> Let $\mathcal{F}$ be a class of functions that ...
```

**Comment by RyotaUshio** [comment]
I will have to somehow directly embed the block ID in `data-callout-metadata` in order to properly display theorem callouts in block transclustions or hover popovers, so this might be harder than I thought.

**Comment by RyotaUshio** [comment]
Somehow I managed to implement it without using the hard-coded index!!

680e0c171

**Comment by RyotaUshio** [comment]
Released in [2.0.0-alpha](https://github.com/RyotaUshio/obsidian-math-booster/releases/tag/2.0.0-alpha)
